app/helpers/sign_requests_helper.rb

```diff
@@ -0,0 +1,20 @@
+module SignRequestsHelper
```

Comment:
Because the data in this helper is intended to represent a page, I would prefer an actual model class as we discussed. I'm personally not a fan of helpers since they can get included in places you don't expect and affect other contexts, but if you'd like to make a case for it, go ahead.